src/test/java/org/apache/commons/configuration2/TestDataConfiguration.java

```diff
@@ -839,7 +842,7 @@ public void testConversionException() throws Exception {
         }
 
         try {
-            conf.get(Class.forName("javax.mail.internet.InternetAddress"), "key1");
+            conf.get(Class.forName("jakarta.mail.internet.InternetAddress"), "key1");
```

Comment:
Another magic string, reuse the constant...

Reply:
This can simply be a direct class reference.